handler.js

```diff
@@ -382,7 +382,8 @@ class GithubBuild {
 
     // deploy and run all tests on release
     // ignore any commands we get
-    if(branch === "release") {
+    // Run on release if push, or pr to release
+    if(branch === "release" || this.enableUatAndFunctionalTests()) {
```

Comment:
Two things:

1. When doing a PR against release: Do we need to execute UAT on PR and push? Would UAT on PR only be enough ? 
2. I changed `enableUatAndFunctionalTests` a little bit, it returns true if we are on PR or if there is a `[on push|pr]` command that matches the current event type. This would break other use cases when opening PR against master or other branches.

Reply:
> When doing a PR against release: Do we need to execute UAT on PR and push? Would UAT on PR only be enough ?

UAT on PR would be enough @mpgo13. 

> I changed enableUatAndFunctionalTests a little bit, it returns true if we are on PR or if there is a [on push|pr] command that matches the current event type. This would break other use cases when opening PR against master or other branches.

will `return this.event.pull_request.base.ref === "release" || parent.enableUatAndFunctionalTests();` cover this case?

Reply:
> will return this.event.pull_request.base.ref === "release" || parent.enableUatAndFunctionalTests(); cover this case?

I'm afraid not, I reckon we need to use `enableUatAndFunctionalTests` as it was meant to be and introduce a new function for handling the `[on push|pr]` command. Would you be happy if I push a commit to this PR ?